pkg/apis/keycloak/v1alpha1/keycloakbackup_types.go

```diff
@@ -7,9 +7,9 @@ import (
 // KeycloakBackupSpec defines the desired state of KeycloakBackup
 // +k8s:openapi-gen=true
 type KeycloakBackupSpec struct {
-	Restore        bool            `json:"restoreFrom,omitempty"`
-	AWS            KeycloakAWSSpec `json:"aws,omitempty"`
-	ExternalAccess KeycloakExternalAccess
+	Restore bool            `json:"restore,omitempty"`
+	AWS     KeycloakAWSSpec `json:"aws,omitempty"`
+	Enabled bool            `json:"enabled,omitempty"`
```

Comment:
Why do we need this? Unless we have a good reason, we should probably revert this change.

Reply:
Good question, not sure why i added this